app/assets/javascripts/analytics/static-analytics.js

```diff
@@ -117,6 +121,19 @@
     });
   };
 
+  StaticAnalytics.prototype.setOptionsForNextPageview = function (options) {
+    if (!GOVUK.cookie || typeof options !== 'object') {
+      return;
+    }
+
+    var cookieOptions = getOptionsFromCookie();
+    $.extend(cookieOptions, options);
+
+    // Singly-stringified JSON sometimes gets escaped when put into a cookie, but inconsistently. The command-line
+    // tests will escape, but browser tests will not. Double-stringify in order to get consistently-escaped strings.
+    GOVUK.cookie('analytics_next_page_options', JSON.stringify(JSON.stringify(cookieOptions)));
```

Comment:
We would probably need to update `/help/cookies` before we can use this live. Is there an alternative to using cookies?